x-pack/test/alerting_api_integration/spaces_only/tests/alerting/builtin_alert_types/index_threshold/alert.ts

```diff
@@ -55,9 +55,6 @@ export default function ruleTests({ getService }: FtrProviderContext) {
       const endDateMillis = Date.now() + (RULE_INTERVALS_TO_WRITE - 1) * RULE_INTERVAL_MILLIS;
       endDate = new Date(endDateMillis).toISOString();
 
-      // write documents from now to the future end date in 3 groups
-      await createEsDocumentsInGroups(3);
```

Comment:
This function creates documents with a timespan field that has values between now and the future.
But the rule looks for last "x" seconds, or in other words: between past and now.
it works, since we use patterns to test the messages. but for the new filterKuery filed i needed an exact number, so i moved this function from beforeEach to each test case. And used a fixed date for the new test case.